rfcs/000-rfc-lightweightRFCProcess.md

```diff
@@ -0,0 +1,87 @@
+
+Some aspects are also based on [Apache Kafka’s KPI process](https://cwiki.apache.org/confluence/display/KAFKA/Kafka+Improvement+Proposals).
+
+All RFCs are submitted via PRs which will be merged when approved.
+
+### Collaboration
+Comments and feedback should be provided on the PR and focus on the technical content. As long as they don't impact the content, collaborators should avoid commenting on formatting, writing style and other maybe relevant, but not critical aspects.
+
+Authors must address all comments written by the deadline. This doesn't mean every comment and suggestion must be accepted and incorporated, but they must be carefully read and responded to. Comments written after the deadline may be addressed by the author, but they should be considered as a lower priority.
+
+Every RFC has a lifecycle. The life cycle has the following phases:
+* **Active**: The deadline for comments on this RFC has passed and the authors have decided to go ahead with it. When making the PR, the doc should start out in this state, since it should be active once the PR is accepted.
+* **Dropped**: The authors have decided not to move forward with the changes proposed in this RFC. This might even happen after the proposal was approved.
+* **Superseded**: The changes proposed on this RFC aren't in effect anymore, the document is kept for historical purposes and there is a new RFC that’s more current.
+
+### Approval
+The proposal should be posted with a date by which the author would like to see the approval decision to be made. How much time is given to comment depends on the size and complexity of the proposed changes. Anything between 1 week and 3 weeks seems like a reasonable timespan. Driving the actual decisions should follow the [lazy majority](https://cwiki.apache.org/confluence/display/KAFKA/Bylaws#Bylaws-Approvals) approach. If a discussion is still going strong once it’s approaching the initially set date, the author(s) might consider extending the time for discussion. This should happen explicitly and be announced on the RFC’s PR by updating the PR description accordingly.
+
+### When to write an RFC?
+Writing an RFC should be entirely voluntary. There is always the option of going straight to a pull request. However, for larger changes, it might be wise to de-risk the risk of rejection of the pull request by first gathering input from the community. Therefore it’s up to every member of our community to decide themselves when they want to reach for this tool.
+
+It’s encouraged to write an RFC for any major change. A major change might be:
+Addition of any major new feature or subsystem
+Changes that impact existing, public APIs. This includes Java APIs, but also things like GFSH commands and externally exposed protocols.
```

Comment:
Add bullets to lines 50 & 51. In the preview of the file they are shown in the same paragraph than line 49.